.cspell.json

```diff
@@ -0,0 +1,171 @@
+{
+    "version": "0.2",
+    "language": "en",
+    "words": [
+      "ukic",
+      "popout",
+      "ICDS",
+      "spacebar",
+      "WCAG",
+      "FOUC",
+      "datepicker",
+      "GCHQ",
+      "Bloggs",
+      "statustext",
+      "helpertext",
+      "whenup",
+      "doubleclick",
+      "Modernizr",
+      "Macchiato",
+      "xxxs",
+      "Rayes",
+      "testspec",
+      "sonarjs",
+      "arrowup",
+      "arrowdown",
+      "arrowleft",
+      "arrowright",
+      "setinput",
+      "validationstatus",
+      "htmlfor",
+      "popperjs",
+      "doubleespresso",
+      "flatwhite",
+      "cappucino",
+      "Ameno",
+      "Aicano",
+      "Menukey",
+      "hidelabel",
+      "apperance",
+      "disableclick",
+      "disablehover",
+      "validationinlineinternal",
+      "Prehydration",
+      "vdom",
+      "hiddeninputvalue",
+      "selectchange",
+      "componentdidload",
+      "Cancelation",
+      "keydown",
+      "autocorrection",
+      "keyup",
+      "mouseleave",
+      "fontsource",
+      "Nunito",
+      "ruleset",
+      "ICDSTEST",
+      "fakepath",
+      "mybuttonid",
+      "viewbox",
+      "interactable",
+      "adipiscing",
+      "elit",
+      "eiusmod",
+      "tempor",
+      "incididunt",
+      "Letraset",
+      "grouplinks",
+      "facilisis",
+      "dignissim",
+      "laoreet",
+      "molestie",
+      "ullamcorper",
+      "Phasellus",
+      "congue",
+      "nisl",
+      "linears",
+      "Pageheader",
+      "Cortado",
+      "Ristretto",
+      "Reallylongonewordanswerthatcouldbeatypeofespresso",
+      "someespresso",
+      "CAPUCCINO",
```

Comment:
Please remove "CAPUCCINO" as it's spelt incorrectly. Correct spelling: "CAPPUCCINO"